sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/inspections/MismatchJoinColumnInspection.kt

```diff
@@ -20,6 +21,16 @@ internal class MismatchJoinColumnInspection : LocalInspectionTool() {
     manager: InspectionManager,
     isOnTheFly: Boolean
   ): Array<ProblemDescriptor> {
+    val projectService = SqlDelightProjectService.getInstance(file.project)
+    if (projectService.module(file.virtualFile)?.let {
+      module ->
+      projectService.fileIndex(module).isConfigured
+    } != true
+    ) {
```

Comment:
Wow this hurt my brain a little to parse the syntax / formatting.

Reply:
i blame spotless it was better when i wrote it

Reply:
I have no doubt. I can see the logic of a fairly unintelligent formatter at work.